vimeo/PlayerKit
# Adds Carthage support to PlayerKit

#### Ticket

[VIM-XXXX](https://vimean.atlassian.net/browse/VIM-XXXX)

- *Note: this is required for Vimeo staff only. If not applicable to your PR, use "N/A".*

#### Pull Request Checklist


#### Issue Summary

Adds Carthage support to the library

#### Implementation Summary

- Remove Pods folder from .gitignore (so the Pods project scheme can be shared)
- Shared `PlayerKit` scheme

#### How to Test

To test simply run `carthage build --no-skip-current`.

Once completed the `Carthage/Build` folder should exist and inside it you will find the compiled framework file under the applicable platform (iOS only in this case).
